src/containers/add-course-banner/AddCourseBanner.styles.ts

```diff
@@ -0,0 +1,27 @@
+export const getStyles = (bannerUrl: string) => ({
+  container: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundSize: 'cover',
+    borderRadius: '10px 10px 0px 0px',
+    px: { md: 7, sm: 3, xs: 3 },
+    py: { md: 6, sm: 4, xs: 4 },
```

Comment:
p: { xs: '4 3', md: '6 7' }

Reply:
There should be no paddings here at all.
Thanks, i deleted them.